src/api/utils/addCardFields.ts

```diff
@@ -34,7 +45,10 @@ export default function addCardFields(
 		const itemPropertyName = getCodeBeamerPropertyNameByFieldLabel(key);
 		try {
 			field = item[itemPropertyName as keyof CodeBeamerItem];
-			if (!field) throw new Error("Field doesn't exist on Item");
+			// if (!field)
+			// 	console.warn(
+			// 		`Field ${itemPropertyName} doesn't exist on Item ${item.name}`
+			// 	);
```

Comment:
Not even a warning? 🤔

Reply:
This seems to be a leftover, which has been altered in the newer branches to
```ts
    try {
		field = item[itemPropertyName as keyof CodeBeamerItem];
		// if (!field)
		// 	console.warn(
		// 		`Field ${itemPropertyName} doesn't exist on Item ${item.name}`
		// 	);
	} catch (error) {
		continue;
	}
```
This is basically expected to throw a fair amount of errors, but that's just how we find out that that Item doesn't have that field.  
I don't explicitly log or display an error / warning (anymore) (and should probably just remove the commented-out block) because this behaviour is communicated in the appCardTagSettings component:  

![image](https://user-images.githubusercontent.com/70019423/192463630-2f7cdb43-172f-4247-af2a-f906779202bf.png)

... which lacks a "Tag" in there.